samples-js/package.json

```diff
@@ -9,8 +9,8 @@
     },
     "dependencies": {
         "@azure/functions": "^4.7.0",
-        "axios": "^1.6.0",
-        "durable-functions": "^3.0.0",
+        "axios": "^1.10.0",
+        "durable-functions": "file:../durable-functions",
```

Comment:
Note: these had to be updated to reference dependencies instead of packages from npm, due to auto-generated package-lock.json containing versions that will flag component governance until release
We can omit these changes, but we will have to release the SDK, update the durable-functions versions, and commit again

Reply:
I see. So, the tradeoff is that the samples can't be copy/pasted and used as-is when we reference the package this way but doing it this way ensures that all samples are using the latest dependencies. I think I'm okay with this.